gen/build_deploy/util.py

```diff
@@ -29,15 +29,15 @@ def copy_makedirs(src, dest):
     shutil.copy(src, dest)
 
 
-def do_bundle_onprem(extra_files, gen_out, output_dir):
+def do_bundle_onprem(gen_out, output_dir):
```

Comment:
Is there no coverage for this? Perhaps next time we touch this code, adding tests/ increasing coverage would be good idea.

Reply:
Unit testing this function wouldn't add much value, it's a helper that doesn't encapsulate much. It'd be one of those tests that's effectively a copy of the function it's testing, but with a bunch of mocks and asserts. It wouldn't do anything to guarantee that the logic in the function is correct with regard to the rest of gen and the installer, and you'd have to tediously update it every time you changed the original code. To really test this function you have to provide it with the output of an onprem genconf run, and then execute a deployment from the resulting `serve/` dir that it creates. Basically you need an integration test.